components/nds-card/scss/card.scss

```diff
@@ -76,6 +76,10 @@
 
   &__summary {
     margin: rem(0 0 $nds-spacing-small 0);
+
```

Comment:
Is this extra line meant to be there?

Reply:
It is yes, as sub selectors always have a space between them and any rules above